examples/funcalign/searchlight_srm_example.py

```diff
@@ -0,0 +1,152 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# Author: Hejia Zhang (Princeton University ELE Department)
+
+"""Distributed Searchlight Example (if run 16 ranks)
+example usage: mpirun -n 16 python3 searchlight_srm_example.py
+"""
+
+import numpy as np
+from mpi4py import MPI
+import sys
+import scipy.io as sio
+from scipy.stats import stats
+from brainiak.searchlight.searchlight import Searchlight
+from brainiak.funcalign.srm import SRM
+import warnings
+
+# parameters
+sl_rad = 1 #searchlight length (of each edge) will be 1+2*sl_rad
+max_blk_edge = 1 #won't change computational results, has effect on MPI processing
+nfeature = 10 #number of features in SRM for each searchlight
+n_iter = 10 #number of interations in SRM
+
+# sanity check
+if sl_rad <= 0 or max_blk_edge <= 0:
+  raise ValueError('sl_rad and max_blk_edge must be positive')
+if nfeature > (1+2*sl_rad)**3:
+  print ('nfeature truncated')
+  nfeature = int((1+2*sl_rad)**3)
+
+# MPI parameters, do not need to change
+comm = MPI.COMM_WORLD
+rank = comm.rank
+size = comm.size
+
+# load data
+movie_file = sio.loadmat('data/sl_movie_data.mat')
+movie_data = movie_file['data']
+
+# Dataset size parameters
+dim1,dim2,dim3,ntr,nsubj = movie_data.shape
+
+# preprocess data, zscore and set NaN to 0
+all_data = [] # first half train, second half test
+for s in range(nsubj):
+  with warnings.catch_warnings():
+    warnings.simplefilter("ignore")
```

Comment:
Yes, this is the warning I want to ignore. There are some 0 voxels in the data which I have to keep, so I will get the warning above when z-scoring the data. I can z-score voxel by voxel and manually check if it is 0, but it would be slower and more complicated. It should be safe to ignore this warning.